layers/+distributions/spacemacs-base/keybindings.el

```diff
@@ -584,10 +584,10 @@ otherwise it is scaled down."
 
 ;; Transparency transient-state
 
-(defun spacemacs/toggle-transparency ()
-  "Toggle between transparent or opaque display."
+(defun spacemacs/toggle-transparency (&optional frame)
+  "Toggle between transparent or opaque display for given or current frame."
```

Comment:
I think it would be better to phrase to docstring similar to how Emacs phrases an optional frame argument. Example from `frame-parameter`:
> Return FRAME’s value for parameter PARAMETER.
> If FRAME is nil, describe the currently selected frame.

In this case, something like:
> Toggle between transparent or opaque display for FRAME.
> FRAME defaults to the currently selected frame.